components/Alert/AlertNotification.tsx

```diff
@@ -0,0 +1,38 @@
+import { Alert as AlertDefault, AlertTitle, AlertDescription } from './Alert';
+import { CheckCircle, XCircle, Info, AlertTriangle } from 'lucide-react';
+
+interface AlertProps {
+  variant?: 'success' | 'error' | 'warning';
+  message: string;
+}
+
+const variantConfig = {
+  success: {
+    title: 'Success!',
+    icon: CheckCircle,
+  },
+  error: {
+    title: 'Error!',
+    icon: XCircle,
+  },
+  default: {
+    title: 'Info',
+    icon: Info,
+  },
+  warning: {
+    title: 'Warning!',
+    icon: AlertTriangle,
+  },
+};
+
+export default function Alert({ variant, message }: AlertProps) {
```

Comment:
By setting this up as the default, you don't need a conditional statement. 
```suggestion
export default function Alert({ variant = default, message }: AlertProps) {
```